From be4d504e27feeb0dbc4a17388fe03628989e4dcb Mon Sep 17 00:00:00 2001
From: Jasper St. Pierre <jstpierre@mecheye.net>
Date: Mon, 29 Aug 2011 21:41:10 +0000
Subject: userMenu: Don't show the default avatar over a newly selected one

When we replace the default avatar image with a real image, we need
to remove the default avatar image.

https://bugzilla.gnome.org/show_bug.cgi?id=657657
---
diff --git a/js/ui/userMenu.js b/js/ui/userMenu.js
index 8d71ce2..33c3315 100644
--- a/js/ui/userMenu.js
+++ b/js/ui/userMenu.js
@@ -226,6 +226,7 @@ IMStatusChooserItem.prototype = {
 
     _setIconFromFile: function(iconFile) {
         this._iconBin.set_style('background-image: url("' + iconFile + '");');
+        this._iconBin.child = null;
     },
 
     _setIconFromName: function(iconName) {
--
cgit v0.9.0.2
